sympy/combinatorics/fp_groups.py

```diff
@@ -544,6 +544,12 @@ def compute_polycyclic_series(self):
 
         return pc_series
 
+    def pc_group(self):
+        '''Returns a the corresponding Pc-Group'''
```

Comment:
"Return a polycyclic presentation of the group, if it exists" - remember the docstring convention: it's always `return` or `compute`, not `returns` or `computes`.